example-monorepos/with-custom-font/apps/next/pages/_document.tsx

```diff
@@ -55,6 +55,8 @@ body {
   -moz-osx-font-smoothing: grayscale;
   -ms-overflow-style: scrollbar;
 }
+
+${customFontCss}
```

Comment:
I forgot this the first time, so I didn't actually fix anything in the last PR 🙃

Reply:
- PR #373